dev.sh

```diff
@@ -6,13 +6,16 @@ DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 
 if [ ! -e "data/config.php" ]; then
-  cp "$DIR/data/config.php.example" "$DIR/data/config.php"
+ cp "$DIR/data/config.php.example" "$DIR/data/config.php"
 fi
 
 if ! which $PHPBIN 2>/dev/null; then
-    PHPBIN=php
+   PHPBIN=php
 fi
 
+# Installing git hook
+./hooks/install.sh
```

Comment:
please use $DIR instead of `.`, since the current directory may not be what you think